### ISSUE TITLE
Coordinates of CaptureDepthPointCloud

### ISSUE THREAD

**Issue by dotchang** [IntelVCL/Open3D#244]
I want to discuss the coordinate system of result of the CaptureDepthPointCloud.
The source code is in the void Visualizer::CaptureDepthPointCloud() of Open3D/src/Visualization/Visualizer/VisualizerRender.cpp.

Similar functions such as CaptureScreenImage and CaptureDepthImage are gives the result based on the sensor(OpenGL Camara) coordinate system.
But, the CaptureDepthPointCloud() function gives pcds that oriented to global coordinates by view_control_ptr_->GetMVPMatrix() of depth_pointcloud.points_.push_back().

I know that it is very useful in usual case. But if I want to make a sensor simulator or evaluation system of  processed point clouds, I have to re-projection or to use CaptureDepthImage function. I think it is little complexity.
I think it would be nice if there is an option that allows the user to give a matrix that changes to view_control_ptr _ -> GetMVPMatrix ().

This is a low priority discussion, if you have time please consider it.

**Comment by qianyizh** [comment]
Good point. I think we can just change the function to:
```
void Visualizer::CaptureDepthPointCloud(const std::string &filename = "",
		bool do_render = true, bool convert_to_world_coordinate = false)
```
The default behavior should be capturing the point cloud in the camera coordinate system, so that it is consistent with ``CaptureDepthImage``. When ``convert_to_world_coordinate`` is set to true, it is the current behavior (using MVPMatrix to convert the point cloud to the world coordinate system).

@dotchang do you mind to submit a PR for this? Thanks.

**Comment by qianyizh** [comment]
Resolved in #247
